documentation/website/src/handbook/integration/serenity-2-migration-guide.md

```diff
@@ -0,0 +1,688 @@
+
+* From your terminal window, clone the repository: `git clone https://github.com/jan-molak/serenity-js.git`
+* Move into the project directory: `cd serenity-js`
+* Update dependencies: `npm install`
+* Build the documentation: `make clean verify report site`
+* Move into the build documentation directory: `cd documentation/website`
+* Serve the documentation: `npm start`
+* Open the documentation in your browser at: `http://localhost:3000`
+* For instance, you'll find this guide [here](http://localhost:3000/handbook/tutorials/serenity-2-migration-guide.html), nestled in the [tutorials](http://localhost:3000/handbook/tutorials/index.html)
+
+## Migrating package.json
+### Updating dependencies
+Various libraries are no longer needed. Here's a high-level summary of changes:
+ * Chai is now transitively brought in via `@serenity-js/assertions` 
+ * mocha is not yet supported so if you used that in 1.x, you'll need to switch to jasmine. 
+ * serenity-cli has been replaced by `@serenity-js/serenity-bdd`. 
+
+So remove all of these from your dependencies section:
+   ```javascript
+    "@types/chai"
+    "@types/chai-as-promised"
+    "@types/mocha"
+    "chai"
+    "chai-as-promised"
+    "chai-smoothie"
+    "mocha",
+    "serenity-cli"
+```    
+Now v2.x is properly published, versions can be set to "latest" if you want to get the latest. 
+However, if you want to want to stay on a fixed version, use that instead.
+
+You can find out the latest available version by visiting the [releases page](https://github.com/jan-molak/serenity-js/releases). :
+   ```javascript
+    "@serenity-js/assertions": "latest",
+    "@serenity-js/console-reporter": "latest",
+    "@serenity-js/core": "latest",
+    "@serenity-js/cucumber": "latest",
+    "@serenity-js/jasmine": "latest",
+    "@serenity-js/protractor": "latest",
+    "@serenity-js/rest": "latest",
+    "@serenity-js/serenity-bdd": "latest",
+    "@types/express": "4.16.1",
+    "@types/jasmine": "3.4.0",
+    "@types/node": "10.14.20",
+    "@types/sinon": "4.3.3",
+    "@types/sinon-chai": "3.2.2",
+    "axios": "0.19.0",
+    "chromedriver": "77.0.0",
+    "jasmine": "3.4.0",
+    "npm-failsafe": "0.4.1",
+    "protractor": "5.4.2",
+    "rimraf": "3.0.0",
+    "ts-node": "8.4.1",
+    "tslint": "5.20.0",
+    "tslint-microsoft-contrib": "6.2.x",
+    "typescript": "3.6.3",    
+ ```   
+
+### Updating scripts
+  
+If you have `--verbose` as an argument in your lint script, remove it as the version of tslint that you now have does not support this parameter. (verbose can now be supplied as a format but stylish is probably better):     
+    
+```javascript    
+"lint": "tslint --config ./tslint.json --project ./tsconfig.json --format stylish",
+```
+         
+Any scripts that used to run `serenity update` or `webdriver-manager update`  should now be updated to `serenity-bdd update`. So for instance:
+
+    "pretest": "serenity-bdd update --ignoreSSL",        
```

Comment:
I tend to do it in `postinstall` rather than `pretest` now.

We might also want to link to https://serenity-js.org/modules/serenity-bdd/
somewhere in this section